src/renderer/store/snackbar.ts

```diff
@@ -0,0 +1,43 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { toast, ToastOptions } from 'react-toastify';
+
+const initialState = {
+  isVisible: false,
+};
+
+const snackbarSlice = createSlice({
+  name: 'snackbar',
+  initialState,
+  // The `reducers` field lets us define reducers and generate associated actions
+  reducers: {
+    showToast: (state) => {
+      state.isVisible = true;
+    },
+    hideToast: (state) => {
+      state.isVisible = false;
+    },
+  },
+});
+
+const { showToast, hideToast } = snackbarSlice.actions;
+
+export const showSnackbar = (message?: string) => (dispatch: any) => {
+  dispatch(showToast());
+  const TOAST_STYLE: ToastOptions = {
```

Comment:
we are also using this config over here - https://github.com/apoorvsadana/madara-tsukuyomi/blob/97cd3d568eb86f49075da6ad2fcebb6a75beb7a9/src/renderer/pages/Apps.tsx#L124

can we store the common things in a config?

Reply:
Fixed this